src/main/java/lotto/domain/LottoResult.java

```diff
@@ -0,0 +1,26 @@
+package lotto.domain;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class LottoResult {
+    private final Map<Rank, Integer> lottoResult = new HashMap<>();
+
+    public LottoResult() {
+        for (Rank rank : Rank.values()) {
+            lottoResult.put(rank, 0);
+        }
+    }
+
+    public void addMatch(int match) {
+        Rank rank = Rank.from(match);
+        if (rank == null) {
+            return;
+        }
+        lottoResult.put(rank, lottoResult.get(rank) + 1);
+    }
```

Comment:
```suggestion
    public void addMatch(int match) {
        addMatch(Rank.from(match));
    }

    public void addMatch(Rank rank) {
        lottoResult.put(rank, lottoResult.get(rank) + 1);
    }
```
생성자와 같이 메서드 오버 로딩을 지원하면 이 API를 개발자 입장에서 편리성을 얻을 수 있지 않을까?